packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -2246,24 +2246,24 @@ Expected response body:
 }
 ```
 
-### Encode_Numeric_Property_uint32AsStringOptional
+### Encode_Numeric_Property_uint8AsStringOptional
 
-- Endpoint: `post /encode/numeric/property/uint32`
+- Endpoint: `post /encode/numeric/property/uint8`
 
-Test operation with request and response model contains property of uint32 type with string encode.
+Test operation with request and response model contains property of uint8 type with string encode.
 Expected request body:
 
 ```json
 {
-  "value": "1"
+  "value": "255"
```

Comment:
why do we want to change it to `uint8`? Would it be better to add a test for `uint8` instead

Reply:
Thanks. I've added back `uint32` test case.